### Instructions

1. It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

* Set up the hypothesis test.
* Write down all the steps followed for setting up the test.
* Calculate the test statistic by hand and also code it in Python. It should be 4.76190. What decision can you make based on this calculated value?

Hint: here is the statistic table

#### Optional
2. In a packing plant, a machine packs cartons with jars. It is supposed that a new machine will pack faster on the average than the machine currently used. To test that hypothesis, the times it takes each machine to pack ten cartons are recorded. The results, in seconds, are shown in the tables in the file Data/machine.txt. Assume that there is sufficient evidence to conduct the t test, does the data provide sufficient evidence to show if one machine is better than the other.


Hint: use two sample t-test

t= (sample_mean(x1) - sample_mean(x2)) / sqrt(square(s1)/n1 + square(s2)/n2 )

### Problem statement
It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. However, In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg

Is the mean of our sample different than 120 mm Hg due to chance, or because the people who took part in the study actually have a mean systolic blood pressure different from 120 mm Hg?

### Process
1. Set up H0 and H1
2. Set critical value CV
3. Compute t score
4. Compare t score vs CV


Significance Level (α): 0.05 (5%)
Chosen test type: one-tailed

Calculate the Test Statistic:
paired t-test
Determine the Degrees of Freedom (df)

This is usually the sample size minus one for a one-sample t-test, and more complex calculations for other types of t-tests.
Find the p-Value:
Using the calculated t-statistic and degrees of freedom, find the p-value from the t-distribution. This can be done using statistical tables, software, or calculators.
Make a Decision:
If the p-value is less than or equal to the significance level (α), reject the null hypothesis in favor of the alternative hypothesis.
If the p-value is greater than α, do not reject the null hypothesis.
Report the Results:
Present the t-statistic, degrees of freedom, p-value, and your conclusion about the hypothesis.
Interpret the Findings:
Discuss what the results mean in the context of the research question.
Remember, the t-distribution is used instead of the normal distribution when the sample size is small (usually less than 30) and the population standard deviation is unknown. The t-distribution is similar to the normal distribution but has fatter tails, allowing for the increased variability that tends to occur in smaller samples.

### Hypothesis
H0: μ <= 120 mm Hg

H1: μ > 120 mm Hg


#### Which distribution should we use?
Our sample size is bigger than 30 but we do not know the poulation standard deviation therefore our sample distribution will follow a a t-student distribution.



In [3]:
import numpy as np
import pandas as pd
import scipy.stats as stats
import matplotlib.pyplot as plt
import seaborn as sns

%matplotlib inline

#### Summary
* population mean μ = 120 mm Hg
* sample mean = 130.1 mm Hg
* sample standard deviation = 21.21 mm Hg
* sample size n = 100

In [26]:
pop_mean = 120
samp_mean = 130.1
samp_std = 21.21
samp_size = 100
signif_level = 0.05
dof = samp_size - 1

In [27]:
# Let's compute the standard error:
sem = samp_std/np.sqrt(samp_size)
sem

2.121

#### T value


In [28]:
# Calculation of t-valu

t_stat = (samp_mean - pop_mean) / (samp_std / np.sqrt(samp_size))
t_stat

4.761904761904759

#### P value

In [29]:
p_value = stats.t.sf(t_stat, dof)
p_value

3.2813509086043083e-06

#### Conclusion
The p value was found to be 3.28 which is higher than our significance Level (α): 0.05
Therfore we cannot reject the null hypothesis that H0: μ <= 120 mm Hg

